apps/fabric-website/src/pages/Controls/SeparatorPage/docs/android/SeparatorImplementation.md

```diff
@@ -4,8 +4,8 @@
 
 ### Source code
 
-[ms_row_divider](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/drawable/ms_row_divider.xml)
+[ms_row_divider](https://github.com/microsoft/fluentui-android/blob/master/FluentUI/src/main/res/drawable/ms_row_divider.xml)
 
 ### Sample code
 
-[ms_row_divider in XML](https://github.com/OfficeDev/ui-fabric-android/blob/master/OfficeUIFabric/src/main/res/layout/view_date_time_picker.xml)
+[ms_row_divider in XML](https://github.com/microsoft/fluentui-android/blob/master/FluentUI/src/main/res/layout/view_date_time_picker.xml)
```

Comment:
This link is broken - I think the control was renamed. Try `view_time_picker.xml` instead of `view_date_time_picker.xml`

Reply:
Thanks for pointing that out :)